project/Dependencies.scala

```diff
@@ -5,7 +5,8 @@ object Dependencies {
     "Local Maven Repo"  at "file://" + Path.userHome.absolutePath + "/.m2/repository",
     "Typesafe Repo"     at "http://repo.typesafe.com/typesafe/releases/",
     "Sonatype releases" at "https://oss.sonatype.org/content/repositories/releases",
-    Resolver.bintrayRepo("mfglabs", "maven")
+    Resolver.bintrayRepo("mfglabs", "maven"),
+    "Elasticsearch releases" at "https://maven.elasticsearch.org/releases/"
```

Comment:
Interesting, nice catch. The last comment in [this](http://stackoverflow.com/questions/28745569/sbt-cant-authorize-in-artifactory#comment46241130_28745569) SO post gave a good clue as to what might be going on. If you append `?trace` to the JAR path you can see the artifact resolution.

Compare the [`releases`](https://maven.elasticsearch.org/releases/org/elasticsearch/plugin/shield/2.2.0/shield-2.2.0.jar?trace) resolution to the [`public-releases`](https://maven.elasticsearch.org/public-releases/org/elasticsearch/plugin/shield/2.2.0/shield-2.2.0.jar?trace) one:

```
2016-03-17T16:53:24.425Z Unable to find resource in releases:org/elasticsearch/plugin/shield/2.2.0/shield-2.2.0.jar
2016-03-17T16:53:24.426Z Searching for info in aggregated repositories
2016-03-17T16:53:24.426Z Preparing list of aggregated repositories to search in
2016-03-17T16:53:24.426Z Appending the virtual repository 'releases'
2016-03-17T16:53:24.426Z Appending collective local repositories
2016-03-17T16:53:24.426Z Appending collective local cache repositories
2016-03-17T16:53:24.426Z Appending collective remote repositories
2016-03-17T16:53:24.426Z Intercepting info request with 'PomInterceptor'
2016-03-17T16:53:24.426Z Intercepting info request with 'MavenMetadataInterceptor'
2016-03-17T16:53:24.426Z Processing request as a release resource
2016-03-17T16:53:24.426Z Searching for the resource within public-releases
2016-03-17T16:53:24.427Z Resource was found in public-releases
```